drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -21,6 +21,16 @@
                       <span class="sr">{{ help_texts.show_title }}</span>
                     </label>
 
+                    <label class="h3" for="problem-mode" title="{{ help_texts.mode }}">{% trans "Problem mode" %}</label>
```

Comment:
@cahrens I wasn't involved in previous a11y review, so I'm afraid I can't answer these questions - basically I've followed an established pattern. I've created a task in our internal backlog to investigate/address this issue.